template/API/Hasura-GraphQL/docker-compose.yaml

```
@@ -0,0 +1,49 @@
+    environment:
+      QUARKUS_LOG_LEVEL: ERROR
+      QUARKUS_OPENTELEMETRY_ENABLED: "false"
+    networks:
+      - hasura-network
+
+volumes:
+  db_data:
+
+networks:
+  hasura-network:
+    driver: bridge
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)